packages/worker/src/sdk/users/users.ts

```diff
@@ -265,6 +280,11 @@ export const save = async (
     await eventHelpers.handleSaveEvents(builtUser, dbUser)
```

Comment:
Just a heads up for learning, we have some similar logic to the above being triggered inside this call:
https://github.com/Budibase/budibase/blob/develop/packages/worker/src/sdk/users/events.ts#L17-L52
see: `unassignAppRoleEvents`

If we were going to continue on the path of sending multiple requests from worker -> server I would recommend to integrate here. However the alternate approach of a single api call to sync as mentioned already I think would be better.

Reply:
I implemented the single app approach